### PR TITLE
feature: CFD-242 Save passenger types

### PR DESCRIPTION
**Pull request by jacsamell**
# Description

-   Save the passenger types to the db

# Testing instructions

-   Go through the passenger type journey and check it gets saved in the DB
-   Go through again with the same passenger category (adult/child) but change a parameter
-   Check that the existing entry gets updated

# Type of change

-   [x] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

# Checklist:

-   [x] Able to run pr locally
-   [x] Followed acceptance criteria
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have made corresponding changes to the documentation if applicable
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tests/pages/api/definePassengerType.test.ts [hunk 1]
```diff
@@ -433,4 +434,24 @@ describe('definePassengerType', () => {
             Location: '/defineTimeRestrictions',
         });
     });
+
+    it('should save the passenger types', async () => {
```
Comment (RobCatton) [inline tests/pages/api/definePassengerType.test.ts:438]: should we have a unit test to say that it doesn't save a group passenger type?

Comment (RobCatton) [reply tests/pages/api/definePassengerType.test.ts:438]: or, should I say, a passenger type which is part of a group

Comment (jacsamell) [reply tests/pages/api/definePassengerType.test.ts]: Good shout, added

File: tests/pages/api/definePassengerType.test.ts [hunk 2]
```diff
@@ -457,16 +457,43 @@ describe('definePassengerType', () => {
...
         const upsertPassengerTypeSpy = jest.spyOn(auroradb, 'upsertPassengerType');
+        const groupPassengerTypesAttribute: GroupPassengerTypesCollection = { passengerTypes: ['adult', 'child'] };
+        const groupSizeAttribute: GroupTicketAttribute = { maxGroupSize: '20' };
+        const mockPreviousPassengerTypeDetails: CompanionInfo[] = [
+            {
+                minNumber: '2',
+                maxNumber: '10',
+                ageRangeMin: '5',
+                ageRangeMax: '10',
+                proofDocuments: ['Membership Card', 'Student Card'],
```
Comment (matt-twomey) [inline tests/pages/api/definePassengerType.test.ts:468]: I don't think it actually affects the test, but users aren't given the option of specifying proofDocuments for adults, so might be more accurate to remove this property?

Comment (jacsamell) [reply tests/pages/api/definePassengerType.test.ts:468]: 👍

**Review by matt-twomey — APPROVED** [review]

**Pull request merged**